### PR TITLE
SDDSIP-1919 - Remove link from and Back button from Feedback page

### PR DESCRIPTION
**Pull request by richard-mohammed-def**
Hides the feedback banner text and back link when on the feedback page.

https://eaflood.atlassian.net/browse/SDDSIP-1919

### CODE REVIEW

**Comment by richard-mohammed-def** [comment]
Test failing is already on develop. Will look into it: 
image

File: packages/web-service/src/pages/feedback/__tests__/feedback.spec.js [hunk 1]
```diff
@@ -54,4 +54,22 @@ describe('feedback.js', () => {
...
+        auth: {
+          credentials: { user: '12345' }
+        }
+      }
+
+      const { getData } = (await import('../feedback.js'))
+
+      const data = await getData(mockRequest)
+
+      expect(data).toEqual({
```
Comment (jonathangoulding) [inline packages/web-service/src/pages/feedback/__tests__/feedback.spec.js:70]: any test for false ?

File: packages/web-service/src/pages/feedback/feedback.js [hunk 2]
```diff
@@ -24,10 +25,18 @@ export const setData = async request => {
   })
 }
 
+export const getData = _request => {
+  return {
+    hideFeedbackBanner: true
```
Comment (jonathangoulding) [inline packages/web-service/src/pages/feedback/feedback.js:30]: not configurable with env ?

File: packages/web-service/src/pages/layout/layout.njk [hunk 3]
```diff
@@ -63,14 +63,18 @@
...
-      },
-      html: '<a class="govuk-link" target="_BLANK" href="/feedback">Your feedback (opens in new tab)</a> can help us to improve this new service.'
-    }) }}
+{% set feedbackBannerHtml = '<a class="govuk-link" target="_BLANK" href="/feedback">Your feedback (opens in new tab)</a> can help us to improve this new service.' %}
+{% if (data.hideFeedbackBanner) %}
+  {% set feedbackBannerHtml = '' %}
+{% endif %}
 
+{% block beforeContent %}
+    {{ govukPhaseBanner({
```
Comment (jonathangoulding) [inline packages/web-service/src/pages/layout/layout.njk:72]: should this be wrapped conditionally instead.

**Pull request merged**